bears/r/RLintBear.py

```diff
@@ -23,7 +23,8 @@ class RLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     REQUIREMENTS = {RscriptRequirement(package='lintr', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+                                       repo='http://cran.rstudio.com',
+                                       version='>=3.1.1')}
```

Comment:
This wont show up anything in bear-docs 
```

        if bear.REQUIREMENTS:
            output += "\n\nDependencies\n------------\n\n"
            outs = []
            for requirement in bear.REQUIREMENTS:
                if hasattr(requirement, 'type'):
                    outs.append("* ``" + requirement.type + "``"
                                " - ``" + requirement.package + "``")
                else:
                    outs.append("* System requirement\n")
                    for system, package in requirement.package.items():
             outs[-1] += "  - ``" + system + "`` - ``" + package + "``"
```
https://github.com/coala/bear-docs/blob/master/generate#L138